README.md

```diff
@@ -1,3 +1,5 @@
+
+Samuel Kalu 
```

Comment:
Hey @eskayML add your name in the name section down below this README.md

Reply:
Okay thanks 👍